src/Security/Authentication/OAuth/ref/Microsoft.AspNetCore.Authentication.OAuth.csproj

```diff
@@ -2,11 +2,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
-    <ExcludeFromSourceBuild>false</ExcludeFromSourceBuild>
```

Comment:
Is this just not necessary since `false` is the default, or do we want to start excluding this?

Reply:
I changed how `$(ExcludeFromSourceBuild)` is set to avoid these numerous special cases.